backend/app/assets/javascripts/spree/backend/promotions.js.coffee

```diff
@@ -26,6 +26,44 @@ window.initProductActions = ->
         $settings.find('input').prop 'disabled', 'disabled'
 
   #
+  # Option Value Promo Rule
+  #
+  if $('#promo-rule-option-value-template').length
+    optionValueSelectNameTemplate = Handlebars.compile($('#promo-rule-option-value-option-values-select-name-template').html())
```

Comment:
We should move these to precompiled Handlebars templates like https://github.com/solidusio/solidus/commit/e757bdb04649d0068c848a29376d83baafc3f0b0 did.